plugins/protocolbuffers/java/v27.1/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/protocolbuffers/java
+plugin_version: v27.1
+source_url: https://github.com/protocolbuffers/protobuf
+description: Base types for Java. Generates message and enum types.
+output_languages:
+  - java
+spdx_license_id: BSD-3-Clause
+license_url: https://github.com/protocolbuffers/protobuf/blob/v27.1/LICENSE
+registry:
+  maven:
+    deps:
+      - com.google.protobuf:protobuf-java:4.27.1
```

Comment:
We can't update this until the upstream Maven artifacts are published: https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java/4.27.1/